### PR TITLE
Fonts exported

### PR DESCRIPTION
**Pull request by emmamarichal**
Hi @EbenSorkin!

I exported the fonts with fontmake, and I don't have fails anymore :)
Here is the QA: [DenkOne_QA.zip](https://github.com/SorkinType/Denk-One/files/10464768/DenkOne_QA.zip)

## Fontbakery report

Fontbakery version: 0.8.10

<details><summary><b>[1] Family checks</b></summary><div><details><summary>ℹ <b>INFO:</b> Check axis ordering on the STAT table.  (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/STAT/axis_order">com.google.fonts/check/STAT/axis_order</a>)</summary><div>

* ℹ **INFO** From a total of 1 font files, 1 of them (100.00%) lack a STAT table.

	And these are the most common STAT axis orderings:
	 [code: summary]
</div></details><br></div></details><details><summary><b>[17] DenkOne-Regular.ttf</b></summary><div><details><summary>⚠ <b>WARN:</b> Check Google Fonts glyph coverage. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage">com.google.fonts/check/glyph_coverage</a>)</summary><div>

* ⚠ **WARN** GF_TransLatin_Pinyin is almost fulfilled. Missing codepoints:

	- 0x01F8 (LATIN CAPITAL LETTER N WITH GRAVE)

	- 0x01F9 (LATIN SMALL LETTER N WITH GRAVE)

	- 0x207F (SUPERSCRIPT LATIN SMALL LETTER N)

	- 0x030D (COMBINING VERTICAL LINE ABOVE)

	- 0x0358 (COMBINING DOT ABOVE RIGHT)
 

	- And 0x1D3A (MODIFIER LETTER CAPITAL N)
 [code: missing-codepoints]
</div></details><details><summary>⚠ <b>WARN:</b> Glyphs are similiar to Google Fonts version? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/production_glyphs_similarity">com.google.fonts/check/production_glyphs_similarity</a>)</summary><div>

* ⚠ **WARN** Following glyphs differ greatly from Google Fonts version:
	* Aacute
	* Agrave
	* Aring
	* Atilde
	* Ccedilla
	* Eacute
	* Egrave
	* Euro
	* Iacute
	* Igrave and 40 more.

Use -F or --full-lists to disable shortening of long lists.
</div></details><details><summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences">com.google.fonts/check/kerning_for_non_ligated_sequences</a>)</summary><div>

* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:

	- f + f

	- f + i

	- i + f

	- f + l

	- l + f 

	- And i + l [code: lacks-kern-info]
</div></details><details><summary>⚠ <b>WARN:</b> Name table strings must not contain the string 'Reserved Font Name'. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/rfn">com.google.fonts/check/name/rfn</a>)</summary><div>

* ⚠ **WARN** Name table entry contains "Reserved Font Name" for a family name ('Denk') that differs from the currently used family name (DenkOne), which is fine. [code: legacy-familyname]
</div></details><details><summary>⚠ <b>WARN:</b> Ensure fonts have ScriptLangTags declared on the 'meta' table. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/meta/script_lang_tags">com.google.fonts/check/meta/script_lang_tags</a>)</summary><div>

* ⚠ **WARN** This font file does not have a 'meta' table. [code: lacks-meta-table]
</div></details><details><summary>⚠ <b>WARN:</b> Font contains '.notdef' as its first glyph? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/mandatory_glyphs">com.google.fonts/check/mandatory_glyphs</a>)</summary><div>

* ⚠ **WARN** Glyph '.notdef' should contain a drawing, but it is empty. [code: empty]
</div></details><details><summary>⚠ <b>WARN:</b> Check font contains no unreachable glyphs (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/unreachable_glyphs">com.google.fonts/check/unreachable_glyphs</a>)</summary><div>

* ⚠ **WARN** The following glyphs could not be reached by codepoint or substitution rules:

	- I.uc

	- firsttonechinesecomb

	- i.cy

	- i.dot

	- i.uc

	- uni0326.loclMAH 

	- And uni1EA2.001
 [code: unreachable-glyphs]
</div></details><details><summary>⚠ <b>WARN:</b> Check if each glyph has the recommended amount of contours. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/contour_count">com.google.fonts/check/contour_count</a>)</summary><div>

* ⚠ **WARN** This font has a 'Soft Hyphen' character (codepoint 0x00AD) which is supposed to be zero-width and invisible, and is used to mark a hyphenation possibility within a word in the absence of or overriding dictionary hyphenation. It is mostly an obsolete mechanism now, and the character is only included in fonts for legacy codepage coverage. [code: softhyphen]
* ⚠ **WARN** This check inspects the glyph outlines and detects the total number of contours in each of them. The expected values are infered from the typical ammounts of contours observed in a large collection of reference font families. The divergences listed below may simply indicate a significantly different design on some of your glyphs. On the other hand, some of these may flag actual bugs in the font such as glyphs mapped to an incorrect codepoint. Please consider reviewing the design and codepoint assignment of these to make sure they are correct.

The following glyphs do not have the recommended number of contours:

	- Glyph name: uni00AD	Contours detected: 1	Expected: 0

	- Glyph name: oslash	Contours detected: 2	Expected: 3

	- Glyph name: aogonek	Contours detected: 3	Expected: 2

	- Glyph name: dcroat	Contours detected: 3	Expected: 2

	- Glyph name: eogonek	Contours detected: 3	Expected: 2

	- Glyph name: Tbar	Contours detected: 2	Expected: 1

	- Glyph name: Uogonek	Contours detected: 2	Expected: 1

	- Glyph name: uogonek	Contours detected: 2	Expected: 1

	- Glyph name: uni0197	Contours detected: 2	Expected: 1

	- Glyph name: ohorn	Contours detected: 3	Expected: 2 

	- And 66 more.

Use -F or --full-lists to disable shortening of long lists.
 [code: contour-count]
</div></details><details><summary>⚠ <b>WARN:</b> Do any segments have colinear vectors? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_colinear_vectors">com.google.fonts/check/outline_colinear_vectors</a>)</summary><div>

* ⚠ **WARN** The following glyphs have colinear vectors:

	* B (U+0042): L<<332.0,1690.0>--<332.0,1690.0>> -> L<<332.0,1690.0>--<560.0,1690.0>>

	* D (U+0044): L<<304.0,1690.0>--<304.0,1690.0>> -> L<<304.0,1690.0>--<599.0,1690.0>>

	* Dcaron (U+010E): L<<304.0,1690.0>--<304.0,1690.0>> -> L<<304.0,1690.0>--<599.0,1690.0>>

	* Dcroat (U+0110): L<<304.0,1690.0>--<304.0,1690.0>> -> L<<304.0,1690.0>--<599.0,1690.0>>

	* Dmacronbelow (U+1E0E): L<<304.0,1690.0>--<304.0,1690.0>> -> L<<304.0,1690.0>--<599.0,1690.0>>

	* Eth (U+00D0): L<<304.0,1690.0>--<304.0,1690.0>> -> L<<304.0,1690.0>--<599.0,1690.0>>

	* P (U+0050): L<<332.0,1690.0>--<332.0,1690.0>> -> L<<332.0,1690.0>--<560.0,1690.0>>

	* R (U+0052): L<<332.0,1690.0>--<332.0,1690.0>> -> L<<332.0,1690.0>--<560.0,1690.0>>

	* Racute (U+0154): L<<332.0,1690.0>--<332.0,1690.0>> -> L<<332.0,1690.0>--<560.0,1690.0>>

	* Rcaron (U+0158): L<<332.0,1690.0>--<332.0,1690.0>> -> L<<332.0,1690.0>--<560.0,1690.0>> 

	* And 21 more.

Use -F or --full-lists to disable shortening of long lists. [code: found-colinear-vectors]
</div></details><details><summary>⚠ <b>WARN:</b> Do outlines contain any jaggy segments? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_jaggy_segments">com.google.fonts/check/outline_jaggy_segments</a>)</summary><div>

* ⚠ **WARN** The following glyphs have jaggy segments:

	* Aogonek (U+0104): B<<884.5,9.5>-<929.0,9.0>-<973.0,9.0>>/B<<973.0,9.0>-<903.0,-7.0>-<878.0,-49.5>> = 12.875001559612462

	* Ccedilla (U+00C7): B<<528.0,-30.5>-<503.0,12.0>-<433.0,28.0>>/B<<433.0,28.0>-<477.0,28.0>-<521.5,28.5>> = 12.875001559612462

	* D (U+0044): L<<280.0,0.0>--<280.0,0.0>>/B<<280.0,0.0>-<206.0,1.0>-<187.5,13.5>> = 0.774220164928034

	* Dcaron (U+010E): L<<280.0,0.0>--<280.0,0.0>>/B<<280.0,0.0>-<206.0,1.0>-<187.5,13.5>> = 0.774220164928034

	* Dcroat (U+0110): L<<280.0,0.0>--<280.0,0.0>>/B<<280.0,0.0>-<206.0,1.0>-<187.5,13.5>> = 0.774220164928034

	* Dmacronbelow (U+1E0E): L<<280.0,0.0>--<280.0,0.0>>/B<<280.0,0.0>-<206.0,1.0>-<187.5,13.5>> = 0.774220164928034

	* Eogonek (U+0118): B<<560.5,9.5>-<605.0,9.0>-<649.0,9.0>>/B<<649.0,9.0>-<579.0,-7.0>-<554.0,-49.5>> = 12.875001559612462

	* Eth (U+00D0): L<<280.0,0.0>--<280.0,0.0>>/B<<280.0,0.0>-<206.0,1.0>-<187.5,13.5>> = 0.774220164928034

	* Iogonek (U+012E): B<<302.5,9.5>-<347.0,9.0>-<391.0,9.0>>/B<<391.0,9.0>-<321.0,-7.0>-<296.0,-49.5>> = 12.875001559612462

	* Scedilla (U+015E): B<<508.0,-30.5>-<483.0,12.0>-<413.0,28.0>>/B<<413.0,28.0>-<457.0,28.0>-<501.5,28.5>> = 12.875001559612462 

	* And 26 more.

Use -F or --full-lists to disable shortening of long lists. [code: found-jaggy-segments]
</div></details><details><summary>⚠ <b>WARN:</b> Do outlines contain any semi-vertical or semi-horizontal lines? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_semi_vertical">com.google.fonts/check/outline_semi_vertical</a>)</summary><div>

* ⚠ **WARN** The following glyphs have semi-vertical/semi-horizontal lines:

	* Eng (U+014A): L<<431.0,1063.0>--<423.0,68.0>>

	* M (U+004D): L<<1120.0,68.0>--<1127.0,1145.0>>

	* M (U+004D): L<<421.0,1145.0>--<426.0,68.0>>

	* N (U+004E): L<<431.0,1063.0>--<423.0,68.0>>

	* Nacute (U+0143): L<<431.0,1063.0>--<423.0,68.0>>

	* Ncaron (U+0147): L<<431.0,1063.0>--<423.0,68.0>>

	* Nmacronbelow (U+1E48): L<<431.0,1063.0>--<423.0,68.0>>

	* Ntilde (U+00D1): L<<431.0,1063.0>--<423.0,68.0>>

	* at (U+0040): L<<198.0,132.0>--<190.0,1078.0>>

	* at (U+0040): L<<414.0,1054.0>--<406.0,132.0>> 

	* And 25 more.

Use -F or --full-lists to disable shortening of long lists. [code: found-semi-vertical]
</div></details><details><summary>ℹ <b>INFO:</b> Show hinting filesize impact. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/hinting_impact">com.google.fonts/check/hinting_impact</a>)</summary><div>

* ℹ **INFO** Hinting filesize impact:

 |               | DenkOne-Regular.ttf          |
 |:------------- | ---------------:|
 | Dehinted Size | 123.3kb |
 | Hinted Size   | 146.3kb   |
 | Increase      | 23.0kb      |
 | Change        | 18.7 %  |
 [code: size-impact]
</div></details><details><summary>ℹ <b>INFO:</b> EPAR table present in font? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/epar">com.google.fonts/check/epar</a>)</summary><div>

* ℹ **INFO** EPAR table not present in font. To learn more see https://github.com/googlefonts/fontbakery/issues/818 [code: lacks-EPAR]
</div></details><details><summary>ℹ <b>INFO:</b> Is the Grid-fitting and Scan-conversion Procedure ('gasp') table set to optimize rendering? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/gasp">com.google.fonts/check/gasp</a>)</summary><div>

* ℹ **INFO** These are the ppm ranges declared on the gasp table:

PPM <= 65535:
	flag = 0x0F
	- Use grid-fitting
	- Use grayscale rendering
	- Use gridfitting with ClearType symmetric smoothing
	- Use smoothing along multiple axes with ClearType®
 [code: ranges]
</div></details><details><summary>ℹ <b>INFO:</b> Check for font-v versioning. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fontv">com.google.fonts/check/fontv</a>)</summary><div>

* ℹ **INFO** Version string is: "Version 1.003; ttfautohint (v1.8.4.7-5d5b)"
The version string must ideally include a git commit hash and either a "dev" or a "release" suffix such as in the example below:
"Version 1.3; git-0d08353-release" [code: bad-format]
</div></details><details><summary>ℹ <b>INFO:</b> Font contains all required tables? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/required_tables">com.google.fonts/check/required_tables</a>)</summary><div>

* ℹ **INFO** This font contains the following optional tables:

	- cvt 

	- fpgm

	- loca

	- prep

	- GPOS

	- GSUB

	- gasp

	- vhea 

	- And vmtx [code: optional-tables]
</div></details><details><summary>ℹ <b>INFO:</b> List all superfamily filepaths (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/superfamily/list">com.google.fonts/check/superfamily/list</a>)</summary><div>

* ℹ **INFO** . [code: family-path]
</div></details><br></div></details>
### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 0 | 11 | 109 | 7 | 100 | 0 |
| 0% | 0% | 5% | 48% | 3% | 44% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **PASS**
* **DEBUG**

### CODE REVIEW

**Comment by EbenSorkin** [comment]
Do you think we should go ahead and publish as is or address a few of the warnings?

**Comment by emmamarichal** [comment]
These are WARN and not FAILS so if there are a few left, it's not a big deal.

`⚠ WARN: Check Google Fonts glyph coverage.` -> You can fill them in if you want but it's not required, it just indicates that only a few characters are missing to complete a language support

`⚠ WARN: Glyphs are similiar to Google Fonts version?` -> This warn is there because I think the glyphs were not composed in the same way (components), but it does not change the design so we can ignore

`⚠ WARN: Font contains '.notdef' as its first glyph?` -> This warn can be avoided by adding the notdef in the glyphset

`⚠ WARN: Name table strings must not contain the string 'Reserved Font Name'.` -> This is not my decision. Normally we don't accept RFN, hence this warn, but we make an exception if the font has already been published with

The rest is about outlines, so can't avoid them because it's about design, but it doesn't matter at all. 
So in conclusion, it's up to you, you can add the notdef and the few remaining glyphs if you want but don't see any obligation :)

**Comment by EbenSorkin** [comment]
I fixed some warnings and caught a few mistakes, and added support for Pan-African Latin And the European language Sami (Latin). I also updated the hyperglot data regarding languages. I am going to re-run the kerning so it includes the new glyphs. And then, if we still get a cupcake on your end when you build in fontmake...let's publish it. :-)

**Comment by EbenSorkin** [comment]
Ok kerning is in!
